pkg/objstore/azure/helpers.go

```
@@ -17,7 +17,7 @@ var (
 const DirDelim = "/"
 
 func getContainerURL(ctx context.Context, accountName, accountKey, containerName string) blob.ContainerURL {
-	c := blob.NewSharedKeyCredential(accountName, accountKey)
+	c, _ := blob.NewSharedKeyCredential(accountName, accountKey)
```

Let's don't ignore errors unless there is explicit reason for it.